main.go

```diff
@@ -23,19 +24,21 @@ func init() {
 	rcommand.AddCommand(listCommand)
 	rcommand.PersistentFlags().StringVarP(&cmd.Host, "server", "s", "myurl", "this is the cluster url")
 	rcommand.PersistentFlags().StringVarP(&cmd.Bearertoken, "token", "t", "usertoken", "this is the user token")
-	rcommand.PersistentFlags().StringVarP(&cmd.Namespace, "namespace", "n", "namespace", "this is the namespace")
+	rcommand.PersistentFlags().StringVarP(&cmd.Namespace, "namespace", "n", "", "this is the namespace")
 	rcommand.MarkFlagRequired("server")
 }
 
 func main() {
+	argsWithProg := os.Args
+	fmt.Println(argsWithProg)
```

Comment:
If this is for better logging, please add some text before/after too.

Also, call this out in the PR description with a sample output.